### PR TITLE
Falcon2 11b

### PR DESCRIPTION
**Pull request by pcuenca**
Continuation of #2076

### CODE REVIEW

**Comment by pcuenca** [comment]
@clefourrier this just fixes a conflict in #2076. Can you please review and merge whenever you need to? Thank you! Also cc @rcojocaru for confirmation that everything's correct.

File: falcon2-11b.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+The second generation of models is focused on increased usability and integrability, building a multi-modal ecosystem. We start this journey by releasing not only the base [11B LLM](https://huggingface.co/tiiuae/falcon-11B), but also the [11B VLM model](https://huggingface.co/tiiuae/Falcon-11B-vlm) that incorporates image understanding capabilities. The vision-language model, or VLM, will allow users to engage in chats about visual content using text.
+
+As with our previous work, the models offer support mainly in English but have good capabilities in ten other languages, including Spanish, French, and German.
+
+## Table of Contents
+
+- [The Falcon2 Models](#the-falcon-models)
+- [11B LLM Training Details](#training)
```
Comment (clefourrier) [inline falcon2-11b.md:48]: Maybe would be clearer by using sub dots, such as
```
- 11B LLM
   - Training details...
   - ...
- 11B VLM
```

File: falcon2-11b.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+|-------------|-------|--------------|--------------|-----------------|----------|--------|----------|-----------|
+| Falcon2-11B | 5500  | 82.91       | 78.30       | 59.73          | 52.56   | 58.37 | 53.83   |  64.28   |
+| Falcon-40B  | 1000  | 85.28       | 81.29       | 61.86          | 41.65   | 56.89 | 21.46   |  58.07   |
+| Falcon-7B   | 1500  | 78.13       | 72.38       | 47.87          | 34.26   | 27.79 | 4.62    |  44.17   |
+| Gemma-7B    | 6000  | 82.47       | 78.45       | 61.09          | 44.91   | 66.03 | 52.77   |  64.29   |
+| Llama3-8B   | 15000 | 82.09       | 77.35       | 59.47          | 43.90   | 66.69 | 44.79   |  62.38   |
+| Mistral-7B  | N/A   | 83.31       | 78.37       | 59.98          | 42.15   | 64.16 | 37.83   |  60.97   |
+
+The Hugging Face Leaderboard team provided an official evaluation of our model on the [Open LLM Leaderboard](https://huggingface.co/spaces/HuggingFaceH4/open_llm_leaderboard) tasks. The model performs better than models such as Llama3-8B (trained on three times more data) and Mistral-7B, and on par with Gemma-7b.
```
Comment (clefourrier) [inline falcon2-11b.md:115]: Yes, but you have almost 2 times the number of params, I'm unsure this comparision is very fair.

File: falcon2-11b.md [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+
+Zero shot performance:
+| Checkpoint  | GT   | HellaSwag | ArcEasy | Winogrande  | ArcChallenge |
+|-------------|------|-----------|----------|------------|--------------|
+| Falcon2-11B | 5500 | 82.07    | 77.78   | 78.30     | 50.17       |
+| Falcon-40B  | 1000 | 82.82    | 81.86   | 76.4      | 54.69       |
+| Falcon-7B   | 1500 | 76.31    | 74.74   | 67.17     | 43.43       | 
+
+The evaluation results show that the Falcon2-11B shows similar performance to Falcon-40B, at a four times smaller model size!
```
Comment (clefourrier) [inline falcon2-11b.md:124]: This is a cool result!

File: falcon2-11b.md [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+|             | *nl*        | 33.6     | 48.7     | 29.8     | 40.0      | 38.0  |
+|             | *ro*        | 32.4     | 44.9     | 29.7     | 37.0      | 36.0  |
+| Bloom-7B    | *de*        | 26.3     |  32.4     | 28.1    | 43.7      | 32.6  |
+|             | *es*        | 38.1     | 56.7     | 28.9     | 40.4      | 41.0  |
+|             | *fr*        | 36.7     | 56.6     | 29.9     | 40.9      | 41.0  |
+|             | *it*        | 29.0     | 40.8     | 27.6     | 43.7      | 35.3  |
+|             | *nl*        | 23.1     | 31.7     | 27.5     | 42.7      | 31.3  |
+|             | *ro*        | 26.9     | 31.8     | 27.4     | 46.1      | 33.1  |
+
+In the spirit of the original Falcon models, the Falcon2-11B was trained not only on English data but also on ten other languages. Our multilingual evaluation results show that the model presents good capabilities in the six languages (*de*, *es*, *fr*, *it*, *nl*, *ro*) featured on the Multilingual LLM Leaderboard and actually shows higher performance than the Falcon-40B and several other multilingual models on all the cited languages.
```
Comment (clefourrier) [inline falcon2-11b.md:163]: Which 10 languages?

**Review by clefourrier — APPROVED** [review]
LGTM, 
Imo, could benefit from a link to the discussion tabs of the model to say "if anything is unclear, feel free to ask further questions at ..."

**Comment by clefourrier** [comment]
@pcuenca Is this PR different from the other one?

**Pull request closed without merging**